src/index.ts

```diff
@@ -65,27 +70,26 @@ async function startClientBot(client) {
   }
 }
 
-function handleCriticalError(error) {
+function handleCriticalError(error: any) {
```

Comment:
Maybe Error is a better interface for this:


```suggestion
function handleCriticalError(error: Error) {
```